.github/workflows/publish-dev-chart.yaml

```diff
@@ -198,3 +198,161 @@ jobs:
+                failed_charts+=("$chart - dependency update failed")
+                continue
+              fi
+
+              if ! helm dependency build $chart; then
+                failed_charts+=("$chart - dependency build failed")
+                continue
+              fi
+
+              if ! helm template $(basename $chart) $chart | kubeconform \
+                -summary \
+                -output json \
+                -schema-location default \
+                -schema-location https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/jsonnet/prometheus-operator/servicemonitors-crd.json \
+                -schema-location https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/jsonnet/prometheus-operator/alertmanagers-crd.json \
+                -schema-location https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/jsonnet/prometheus-operator/prometheusrules-crd.json \
+                -schema-location https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/jsonnet/prometheus-operator/prometheuses-crd.json \
+                -schema-location https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/jsonnet/prometheus-operator/podmonitors-crd.json \
+                -strict \
+                -; then
+                failed_charts+=("$chart - validation failed")
+              fi
+            fi
+          done
+          
+          if [ ${#failed_charts[@]} -ne 0 ]; then
+            echo "faild charts are:"
+            printf '%s\n' "${failed_charts[@]}"
+            exit 1
+          fi
+
+  run-k8s-test:
+    name: check-services-up-and-running
+    runs-on: ubuntu-latest
+    needs: 
+      - publish-ghcr-scroll-sdk
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.12.1
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.6.0
+
+      - name: Replace helm registry by helm/dev
+        run: |
+          find devnet/ -type f -exec sed -i 's|oci://ghcr.io/scroll-tech/scroll-sdk/helm|oci://ghcr.io/scroll-tech/scroll-sdk/helm/dev|g' {} +
+
+      - name: Create Kind Cluster
+        uses: helm/kind-action@v1.8.0
+        with:
+          cluster_name: chart-testing
+
+      - name: Helm registry login
+        run: |
+          helm registry login ghcr.io/scroll-tech/helm/scroll-sdk --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run chart-testing (install)
+        if: steps.list-changed.outputs.changed == 'true'
+        run: ct install --config ct.yaml --charts scroll-sdk --target-branch ${{ github.event.repository.default_branch }}
```

Comment:
what os the purpose of this command ?
You are running a make install a few step below, are you installing 2 times the chart ?

Reply:
You are right, this is a leftover from the previous solution and should be removed.